book/07-extending-learners.qmd

```diff
@@ -102,166 +108,190 @@ For our example, the resulting script looks like this:
+    * `distr` - Predicted survival distribution, either discrete or continuous.
+      Implemented in `r mlr3book::cran_pkg("distr6")`.
+    * `crank` - Continuous risk ranking.
+    * `response` - Predicted survival time.
+  * `r ref("LearnerDens")`
+    * `pdf`- Predicts the probability density function.
+    * `cdf` - Predicts the cumulative distribution function.
+    * `distr` - Predicts a distribution as implemented in `r mlr3book::cran_pkg("distr6")`.
+  * `r ref("LearnerClust")`
+    * `partition` - Assigns the observation to a partition.
+    * `prob` - Returns a probability for each partition.
 * `feature_types`: Set of feature types the learner is able to handle.
   See `r ref("mlr_reflections", text = "mlr_reflections$task_feature_types")` for feature types supported by `r mlr3book::mlr_pkg("mlr3")`.
 * `properties`: Set of properties of the learner. See `r ref("mlr_reflections", text = "mlr_reflections$learner_properties")` for the full list of feature types supported by `r mlr3book::mlr_pkg("mlr3")`. Possible properties include:
   * `"twoclass"`: The learner works on binary classification problems.
   * `"multiclass"`: The learner works on multi-class classification problems.
   * `"missings"`: The learner can natively handle missing values.
   * `"weights"`: The learner can work on tasks which have observation weights / case weights.
-  * `"parallel"`: The learner supports internal parallelization in some way.
-    Currently not used, this is an experimental property.
   * `"importance"`: The learner supports extracting importance values for features.
     If this property is set, you must also implement a public method `importance()` to retrieve the importance values from the model.
-  * `"selected_features"`: The learner supports extracting the features which were used.
+  * `"selected_features"`: The learner supports extracting the features which were selected by the model.
     If this property is set, you must also implement a public method `selected_features()` to retrieve the set of used features from the model.
+  * `"oob_error"`: The learner supports extracting the out of bag error.
+    If this property is set, you must also implement a public method `oob_error()` to retrieve the out of bag error from the model.
+  * `"loglik"`: The learner supports extracting the log-likelihood of the learner.
+    If this property is set, you must also implement a public method `loglik()` to retrieve the log-likelihood from the trained model.
+  * `"hotstart_forward"`: The learner allows to continue training the model e.g. by adding more trees to a random forest.
+    If this property is set, you must also implement a private method `.hotstart()`.
+  * `"hotstart_backward"`: The learner allows to "undo" some of the training, e.g. by removing some trees from a model.
+    If this property is set, you must also implement a private method `.hotstart()`.
```

Comment:
duplicated

Reply:
removed it from the instructions in the beginning